### ISSUE TITLE
Rule Request: multiline_arguments: Allow enforcing only after first closure

### ISSUE THREAD

**Issue by ldiqual** [realm/SwiftLint#1896]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule Request

My proposal is to add a new configuration for `multiline_arguments` that would allow enforcing the rule **only after the first closure**.

For instance, I would like to handle this case:

```swift
UIView.animate(withDuration: duration, animations: {
    // animations
}, completion: { _ in
    // completion
})
```

However with the current defaults, I have to write it this way:

```swift
UIView.animate(
    withDuration: duration,
    animations: {
        // animations
    }, completion: { _ in
        // completion
    }
)
```

You would also be able to pass more inline arguments before `animations`.

 >1. Why should this rule be added? Share links to existing discussion about what
   the community thinks about this.

I believe that the animate + completion case above is common across iOS codebases and that the first style above is a common way to express it.

Ray Wenderlich's swift style guide passes `animations` on the same line: https://github.com/raywenderlich/swift-style-guide#closure-expressions

 >2. Provide several examples of what _would_ and _wouldn't_ trigger violations.

**Doesn't trigger**

```swift
call(a: a, b: {

}, c: {

})
```

```swift
call(a: a, b:b, c: {

}, d: {

})
```

**Triggers**

```swift
call(a: a, b:b,
  c: c, d: {

  }, d: {

  }
)
```

```swift
call(a: a, b: {

}, c: {

}, d: d, e: e)
```

 >3. Should the rule be configurable, if so what parameters should be configurable?

A name for such option could be:

 - `allow_first_closure_on_same_line`
 - `only_enforce_after_first_closure`

Hard to describe the intent in a generic way and I'd love input from you guys.

 >4. Should the rule be opt-in or enabled by default? Why?
   See [README.md](../README.md#opt-in-rules) for guidelines on when to mark a
   rule as opt-in.

The option would be enabled by default for the `multiline_arguments` rule, since this is a common case as described above.

**Comment by tinder-garricnahapetian** [comment]
I think this is a great idea. Would it be hard to implement? I could give it a try. Where would I start?

**Comment by jpsim** [comment]
Shouldn't be too hard. Start in [MultilineArgumentsRule.swift](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/MultilineArgumentsRule.swift) and [MultilineArgumentsConfiguration.swift](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/MultilineArgumentsConfiguration.swift).

**Comment by hani-ibrahim** [comment]
any updates on this?

**Comment by maciesielka** [comment]
Took a stab at adding this in #2281

**Comment by marcelofabri** [comment]
Implemented by @maciesielka in #2281 🎉
